packages/marketplace/src/selector/wehooks.ts

```diff
@@ -19,3 +19,7 @@ export const selectTopicsLoading = (state: ReduxState) => {
 export const selectApplicationId = (state: ReduxState) => {
   return state?.webhooks?.topics?.applicationId
 }
+
+export const selectWebhookEditModalType = (state: ReduxState) => {
+  return state.webhookEdit.modalType
```

Comment:
To avoid some unexpected problem I think we need to change
```
state.webhookEdit?.modalType
```

Reply:
webhookEdit was defined as a required property in ReduxState interface, using optional chaining is not necessary here